Misc/NEWS.d/next/Core and Builtins/2021-10-23-00-39-31.bpo-45582.YONPuo.rst

```diff
@@ -0,0 +1,3 @@
+Path calculation (known as ``getpath``) has been reimplemented as a frozen
+Python module. This should have no visible impact, but may affect
+calculation of all paths referenced in :mod:`sys` and :mod:`sysconfig`.
```

Comment:
That's a rather vague caveat. 🙂

Reply:
I'm trying to keep my NEWS entries focused on the user who is reading it. If they notice anything, it'll be those paths, and then this description will get them back to the bug and a place to discuss it.

Ultimately, it's impossible to predict what weird behaviours people have set up and will get to see once this lands. Trying to write out the list will just be unnecessarily scary, and necessarily incomplete, because 99.9% of people use the default install layout and will be fine.

Reply:
Agreed.